www/src/pages/overview/developers.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Developers
+description: How to code with Thumbprint.
+---
+
+import { Img } from 'components/mdx';
+
+Thumbprint provides code in a number of technologies. React and SCSS components, [design tokens](https://medium.com/eightshapes-llc/tokens-in-design-systems-25dd82d58421), mixins, atomic CSS, and custom SCSS are all available as tools to build UIs. But knowing what to use and when to use them isn’t always clear, especially to those new to Thumbprint. We’ve put together this document to help guide those decisions.
+
+## React projects
+
+### RATS
+
+No, not the little animal. It’s an acronym to help remember the order of technologies to try.
+
+1. **R**eact
+2. **A**tomic
+3. **T**okens
+4. **S**ass
```

Comment:
Not sure if I understand this one.

Does it mean custom CSS in the developer's own project? Or our Thumbprint SCSS components?

If it's custom CSS in the developer's project, then this could probably merged into the description for the "Tokens" section since tokens are almost always used in custom CSS files.

Reply:
I do see them as separate topics though maybe it's not clear enough. 

Tokens should be separate because we make them available in JS they are used occasionally:

``
import {
    tpColorBlue,
    tpColorBlack300,
    ...
}
```

By "custom CSS", which is what I mean by the fourth item, I mean token-less Sass. I can drop the acronym and make it clearer that...

Tokens = JS tokens in JSX and Sass tokens in Sass modules
Custom Sass/CSS =  hardcoded values where no token is appropriate

Reply:
> By "custom CSS", which is what I mean by the fourth item, I mean token-less Sass. I can drop the acronym and make it clearer that...

Ah, ok. Yeah, I think that can be clarified. I thought it meant Sass with tokens.

Reply:
Updated to be clearer.